build/bazel/remote/execution/v2/remote_execution.proto

```diff
@@ -1288,6 +1315,218 @@ message WaitExecutionRequest {
   string name = 1;
 }
 
+// A request message for
+// [ActionCache.GetPotentialActionResults][build.bazel.remote.execution.v2.ActionCache.GetPotentialActionResults]
+// two-phase cache lookups.
+message GetPotentialActionResultsRequest {
+  // The instance of the execution system to operate against. A server may
+  // support multiple instances of the execution system (with their own workers,
+  // storage, caches, etc.). The server MAY require use of this field to select
+  // between them in an implementation-defined fashion, otherwise it can be
+  // omitted.
+  string instance_name = 1;
+
+  // The hash of an [Action][build.bazel.remote.execution.v2.Action] that acts as a weak fingerprint
+  // key to look up potential action results that must be compared to the client's current filesystem
```

Comment:
Env var lookups as addition to key?

Reply:
(to weak fingerprint)?

---
In reply to: [453832090](https://github.com/erikma/remote-apis/pull/1#discussion_r453832090) [](ancestors = 453832090)

Reply:
Already in the Command as a concept - that has no exact spec as to which envv vars are included in the command fingerprint.

---
In reply to: [453832241](https://github.com/erikma/remote-apis/pull/1#discussion_r453832241) [](ancestors = 453832241,453832090)